index.js

```diff
@@ -39,35 +38,13 @@ async function mercuriusDynamicSchema(fastify, opts) {
       async childServer => {
         childServer.register(mercurius, {
           schema: schema.schema,
+          path: schema.path,
           resolvers: schema.resolvers,
           graphiql: false,
-          routes: false
-        })
-
-        childServer.route({
-          path: schema?.path ?? '/graphql',
-          method: 'POST',
-          constraints: { [STRATEGY_NAME]: schema.name },
-          handler: async (req, reply) => {
-            let { query, operationName, variables } = req.body
-
-            if (typeof req.body === 'string') {
-              query = req.body
-            }
-
-            if (contextFn) {
-              req[kRequestContext] = await contextFn(req, reply)
-              Object.assign(req[kRequestContext], { reply, childServer })
-            } else {
-              req[kRequestContext] = { reply, app: childServer }
-            }
-
-            return reply.graphql(
-              query,
-              req[kRequestContext],
-              variables,
-              operationName
-            )
+          context: contextFn,
+          routes: true,
+          additionalRouteProps: {
```

Comment:
should be `additionalRouteOptions`, I don't have push access to the repo anymore

Reply:
ah thanks, good catch 👌